Resources/ServerInfo/_NF/Guidebook/Shipyard/Bulker.xml

```diff
@@ -81,5 +87,8 @@
   
   ## Sidenotes
   
-  * - Bulker-class salvage ships are equipped with two APC units that can be used to appraise the ship's total power consumption. Unmodifued Bulker-class ship requires 24.9 kW of power to remain operational.
+  * - Bulker-class salvage ships are equipped with two APC units that can be used to appraise the ship's total power consumption. Unmodifued Bulker-class ship requires 26.6 kW of power to remain operational.
+  
+  ** - Bulker-class salvage ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, so the standard P.A.C.M.A.N. was replaced with S.U.P.E.R.P.A.C.M.A.N.: recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 28 kW. Furthermore.
```

Comment:
You seem to have trailed off.
```suggestion
  * - Bulker-class salvage ships are equipped with two APC units that can be used to appraise the ship's total power consumption. Unmodified Bulker-class ships require 26.6 kW of power to remain operational.
  
  ** - Bulker-class salvage ships have higher power demand than a standard P.A.C.M.A.N. can provide at optimal settings, so a S.U.P.E.R.P.A.C.M.A.N. was installed instead. Please note that the recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 28 kW.
```